config/config-server/test/com/thoughtworks/go/config/MagicalGoConfigXmlWriterTest.java

```diff
@@ -382,6 +383,13 @@ public void shouldNotWriteMoreThanOneOnCancelTaskWhenDefined() throws Exception
     }
 
     @Test
+    public void shouldBeAValidXSD() throws  Exception {
+        SchemaFactory factory = SchemaFactory.newInstance("http://www.w3.org/2001/XMLSchema");
+        File schemaLocation = new File("resources/cruise-config.xsd");
+        factory.newSchema(schemaLocation);
```

Comment:
Change to `factory.newSchema(new StreamSource(getClass().getResourceAsStream("/cruise-config.xsd")));` please. Files in the resource directory are in the classpath.